crates/driver/src/infra/config/file/mod.rs

```diff
@@ -1,6 +1,6 @@
 pub use load::load;
 use {
-    crate::{domain::eth, util::serialize},
+    crate::{domain::eth, infra::persistence::S3, util::serialize},
```

Comment:
Although it often seems like unnecessary boiler plate there should be a separate struct that defines the config format which get used to initialize the s3 struct.
That's the reason why the new config struct is the only thing that gets imported here. All the other config formats have been defined inside this file and therefore didn't need to be imported.
This separation between `config representation` <-> `struct used in the core logic` <-> `dto` allows us to very freely modify each component without propagating this change down the whole chain. Because we didn't do that diligently in the old code modifying anything of the `Order` in the `api` is extremely annoying.